batch_object_tagger.tf

```diff
@@ -166,3 +166,36 @@ resource "aws_batch_job_definition" "s3_object_tagger" {
   }
   CONTAINER_PROPERTIES
 }
+
+resource "aws_batch_job_definition" "s3_object_tagger_test_ami" {
```

Comment:
These should only be deployed where the tests will be carried out, something like:

```
   count = local.environment == "development" || local.environment == "qa" ? 1 : 0
```

Remember that you will need to use the count when this is called elsewhere ie `aws_batch_job_definition.s3_object_tagger_test_ami[0]`